quickget

```diff
@@ -263,7 +263,8 @@ function editions_arcolinux() {
 }
 
 function releases_blendos() {
-    echo 23.01
+    curl -s https://api.github.com/repos/blend-os/blendOS/releases  |grep tag_name | grep -o -E '[[:digit:]]+\.[[:digit:]]+'
```

Comment:
@RudraSwat does this look reasonably sane for catching your future release tags so QuickEmu can pick them up dynamically ?

Reply:
@philclifford yep, lgtm +1 (I'll be sticking to this format for future release tags)